### ISSUE TITLE
Directus all API endpoints are returning 404 error

### ISSUE THREAD

**Issue by Lalarukh-khan** [directus/directus#23082]
### Describe the Bug

I have uploaded my directus code into the ssh server. But none of my APIs endpoint whether POST or GET is working, all returning 404 errors. However I can see all collection, all items, can login etc in frontend.

I am running my frontend in: http://platform.groep-e.be:8080/

And my backend is running on: http://platform.groep-e.be:8055/

Followed this article: https://docs.directus.io/blog/deploy-directus-ubuntu-server.html. Am I missing something over association of APIs endpoint in backend?

In the localhost the project is working totally fine, all endpoints working. I don't know what's either missing or wrong that all API endpoints are responding: `route not found`.

One of the GET request for testing: `curl http://platform.groep-e.be:8055/akeneoproducts`

How to make all endpoints in backend of Directus to respond and work.

### To Reproduce

Need guidance.

### Directus Version

10.13.1

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by hanneskuettner** [comment]
This is more of a setup question rather than a bug in Directus itself and must ask you to direct those to our [Community Discord](https://chat.directus.io) as GitHub is the wrong place for those.

That being said, it seems like you've not followed the article step for step, as the resulting setup should run on port 80 and not on two different ports. Also having had a quick look your API is actually running on port 8080 on that domain. Please ask any follow-up questions in our Discord. Thank you!

**Comment by Lalarukh-khan** [comment]
Hey @hanneskuettner 
I added my concern in the discord community but couldn't get suitable guidance or follow up response. 

Secondly, frontend is not the concern for me ATM. By following that article I got my project to run on the port 8055. As we can see in the article as well. 
The thing I am seeking guidance is for that those custom APIs developed manually by me in the controller are running smoothly in localhost then why not in SSH server. 
When I'm running this command for localhost, it respond back with 200: `curl http://localhost:8055/akeneoproducts`

But when the same I run in the docker of SSH server, its returning route not found however its present there: `curl http://platform.groep-e.be:8055/akeneoproducts`

Frontend is not the game, that is deployed by me individually. 
But why endpoint APIs are not working in backend in SSH?
